tigerpath/majors_and_certificates/majors/COM_2018.json

```
@@ -0,0 +1,263 @@
+      "course_list": [
+        "LANG 107"
+      ]
+    },
+    {
+      "name": "Introductory Courses",
+      "max_counted": null,
+      "min_needed": null,
+      "explanation": "Students who wish to concentrate in comparative literature are advised (though not required) to take COM 205-206, HUM 216-219 or TRA 200 in their sophomore year or earlier.",
+      "double_counting_allowed": true,
+      "completed_by_semester": 4,
+      "course_list": [
+        "COM 205",
+        "COM 206",
+        "HUM 216",
+        "HUM 217",
+        "HUM 218",
+        "HUM 219",
+        "TRA 200"
+      ]
+    },
+    {
+      "name": "Departmental Courses",
+      "max_counted": 1,
+      "min_needed": "ALL",
+      "explanation": "Students in Comparative Literature select courses from a wide range of offerings throughout the university and are encouraged to construct a program of study to match their individual interests. Nine departmental courses are required of each student, chosen according to the type of comparative work pursued. These courses must be taken for a grade; no P/D/F course can be counted as a departmental requirement, with two exceptions: for the track courses only, you may use approved courses that are not offered for a grade, such as CWR and THR courses; or the Historical Requirement course, if it is not one of the nine departmentals. (Although, do note that ungraded courses cannot raise your grade point average, if that is needed.) The three COM courses and four reading courses must be graded. The three main rubrics and various tracks are listed below, as well as a description of the Historical Requirement.",
+      "double_counting_allowed": false,
+      "req_list": [
+        {
+          "name": "Comparative Literature",
+          "max_counted": 1,
+          "min_needed": 2,
+          "explanation": "All majors beginning with the class of 2021, must take COM 300 (the Junior Seminar), and two other COM courses taught by core COM faculty (that is, by faculty listed on the “Core Faculty” list of the departmental website). Majors must take COM 300 in the fall term of their junior year, unless they are studying abroad then, in which case they must take the course in their senior year instead. COM has also sometimes accepted the English-department junior seminar taught in London by Princeton faculty. The other two COM courses can be 200-, 300-, 400-, or 500-level; with the professor’s permission, advanced students may enroll in 500-level COM courses. COM majors in the classes of 2019 and 2020 fulfill this requirement by taking COM 300 (the Junior Seminar), and any other COM courses or courses cross-listed with COM. These must be Princeton COM courses; no exceptions.",

```

Can remove the instructions for classes of 2019 and 2020. No longer relevant.
```suggestion
          "explanation": "All majors beginning with the class of 2021, must take COM 300 (the Junior Seminar), and two other COM courses taught by core COM faculty (that is, by faculty listed on the “Core Faculty” list of the departmental website). Those other two COM courses can be 200-, 300-, 400-, or 500-level; with the professor’s permission, advanced students may enroll in 500-level COM courses. These must be Princeton COM courses; no exceptions.",
```